### ISSUE TITLE
Fatal error: [Error: Required  not provided]

### ISSUE THREAD

**Issue by errolgr** [jscrambler/grunt-jscrambler#23]
Error when installing grunt-jscrambler versions greater than 0.2.10 with node v4.4.3.

The error is resolved when downgrading jscrambler back to 0.2.10.

**Comment by optimalisatie** [comment]
I am experiencing the same issue with grunt-jscrambler `v1.0.2` and node `v6.3.0`

**Comment by swateek** [comment]
Same issue even with version 0.2.10 and node version 4.4.2

**Comment by magalhas** [comment]
Can you please show me your task configuration object? That message should be a missing access or secret keys.

**Comment by swateek** [comment]
Yes, my task was missing access and secret keys. They were blank. Cannot purchase jscrambler either, any other alternative?

**Comment by s-coimbra21** [comment]
@swateek Unfortunately, Jscrambler has no viable free alternatives. However, you may find solace in knowing that Jscrambler has a 30 day free trial and its fully featured **Professional** edition can be purchased for as low as 85$ per month. 

For more information visit https://jscrambler.com/

Thank you and have a very nice day!

**Comment by magalhas** [comment]
You can also use some free features forever.

**Comment by runningbug** [comment]
I am also getting this error using grunt-jscrambler 1.0.2. Works fine using version 0.2.10.
